charts/cert-manager-issuers/tests/__snapshot__/awspcaclusterissuers_test.yaml.snap

```diff
@@ -1,14 +1 @@
-should create a AWSPCAIssuer:
-  1: |
-    apiVersion: awspca.cert-manager.io/v1beta1
-    kind: AWSPCAClusterIssuer
-    metadata:
-      labels:
-        app.kubernetes.io/app: cert-manager-issuers
-        app.kubernetes.io/instance: RELEASE-NAME
-        app.kubernetes.io/managed-by: Helm
-        app.kubernetes.io/version: "1"
-        helm.sh/chart: cert-manager-issuers-0.3.0
-      name: example
-    spec:
-      arn: <some-pca-arn>
+{}
```

Comment:
It looks like the snapshot updater did an upsi wupsi... please revert all changes that aren't in the keycloak-operator chart

Reply:
Done :+1: